src/logic/Utils.ts

```diff
@@ -186,16 +186,41 @@ export const generateSchemesOptions = (schemes: SchemeIni) => {
   ));
 };
 
-// Reset any Marketplace localStorage keys (effectively resetting it completely)
-export const resetMarketplace = () => {
+/**
+ * Reset Marketplace localStorage keys
+ * @param category The category to reset. If not provided, reset all categories.
+ */
+export const resetMarketplace = (category: "extensions" | "snippets" | "theme" | "all" = "all") => {
```

Comment:
I think it should be possible to make it accept multiple categories using spread operator.
Just in case like they want to reset 2/3 categories and shouldn't need to type twice.

Reply:
Updated.